src/js/program/background-media.ts

```diff
@@ -12,34 +18,36 @@
  * @property {Promise} nextMedia - A promise representing the media loading in the background. Resolves with the mediaController.
  * @constructor
  */
-export default function BackgroundMedia() {
-    let currentMedia = null;
-    let nextMedia = null;
+export default function BackgroundMedia(): object {
```

Comment:
I think it might be worth making a `BackgroundMedia` type to use here instead of `object`.